.github/workflows/moodle-plugin-ci.yml

```diff
@@ -139,7 +109,7 @@ jobs:
 
       - name: Validating
         if: ${{ always() }}
-        run: moodle-plugin-ci validate
+        run: moodle-plugin-ci validate || true
```

Comment:
Hi @lucaboesch Why are we ignoring the outcome of the `moodle-plugin-ci validate` command? Just trying to understand the logic behind this particular change :smile: